network-api/networkapi/wagtailpages/__init__.py

```diff
@@ -16,3 +20,54 @@ def register_howto_menu_item():
         _('How Do I Wagtail'), reverse('how-do-i-wagtail'),
         name='howdoIwagtail', classnames='icon icon-help', order=900
     )
+
+
+# WARNING: this is not necessarily a good idea, but is the only way to override
+# Django's default behaviour of requiring language codes to be lowercased.
+# We have to modify the core Django method, because we have no way to replace
+# all the core functionality that relies on this - e.g., url resolvers that
+# the Django admin and third party apps use.
+# This is pretty ugly, and ideally this should be fixed upstream.
```

Comment:
Do we have a link to the issue on the Django side for this? If there isn't one, we should probably file one, especially if MoCo's run into this problem too.

Reply:
I don’t think we have one, unless Torchbox filed one when they wrote this. I can check with MoCo folks too.
I often stumbled upon Django tickets when looking for i18n-related stuff I needed, managed to find the exact ticket covering my needs, just to see it closed as WONTFIX :S But we can try!

Reply:
that's fine, as long as we have something to point to